go/cmd/vttablet/vttablet.go

```diff
@@ -103,7 +104,10 @@ func main() {
 	// If connection parameters were specified, socketFile will be empty.
 	// Otherwise, the socketFile (read from mycnf) will be used to initialize
 	// dbconfigs.
-	config.DB = config.DB.Init(socketFile)
+	config.DB.Init(socketFile)
+	for _, cfg := range config.ExternalConnections {
+		cfg.Init("")
```

Comment:
nit - Maybe it would be clearer to have `InitWithSocket` and `Init()`  I found it a bit confusing that this is initialized with empty string without any context on why that's the case.